analyticConfigurations/metric_meta-dynamodb.json

```diff
@@ -12,10 +12,10 @@
         }
       },
       {
-        "match": "aws\\.dynamodb\\..*units",
+        "match": "aws\\.dynamodb\\.provisioned(read|write)capacityunits",
```

Comment:
Should this sparse strategy be applied to `consumed(read|write)capacityunits`, too?

Reply:
No, the consumed and provisioned have different behaviors. Provisioned is mostly stagnant as it's a value set by a user. It can be set to autoscale but even then the values don't have much variability. Consumed is constantly changing as read/write operations are performed against the DB.